api/routes/ticket.js

```diff
@@ -0,0 +1,17 @@
+var express = require('express');
+var router = express.Router();
+var mongoose = require('mongoose');
+var Ticket = require('../models/Ticket');
+
+//Save Ticket 
```

Comment:
```suggestion
// Save Ticket 
```

Generally, there is a space before the beginning of a comment.